packages/tools/components-package/nps.js

```diff
@@ -55,7 +58,7 @@ const getScripts = (options) => {
 			default: "nps prepare lint build.bundle",
 			templates: `mkdirp dist/generated/templates && node "${LIB}/hbs2ui5/index.js" -d src/ -o dist/generated/templates`,
 			styles: {
-				default: "nps build.styles.themes build.styles.components",
+				default: `nps build.styles.themes build.styles.components ${copySrcGenerated}`,
```

Comment:
In JS mode after the styles` JS files are generated into src/generated, they are not being copied into dist. Because the general copy command is parallel to build styles command., so either we need to reshuffle the commands or as here - add one more copy task that runs only in JS mode and over the src/generated folder.